src/Orleans/Serialization/SerializationManager.cs

```diff
@@ -527,18 +622,15 @@ public static void Register(Type type, Type serializerType)
                 {
                     Register(
                         type,
-                        obj =>
-                        {
+                        obj => {
```

Comment:
no worries, but do you mind reverting these? I think it might cause unnecessary conflicts with our core clr branch, where serialization is a big conflicting point